JavaScript_json_class/image_object_class.js

```diff
@@ -0,0 +1,196 @@
+/*
+Structure of image Json object:
+
```

Comment:
We will want to add the starting point of the box, along with the coordinates for the box such as x1, x2, y1, and y2.  We can then either do the math to figure out the width/height or just have another field for it.